app-template/src/Navbar.tsx

```diff
@@ -250,37 +189,32 @@ export class Navbar extends React.Component<{}, NavbarState> {
                   if (el.isNested) {
                     // handle the case when the menu has sub options
                     const plainRoutes = el.routes as GenericRoute []
-                    const key = `dropdown${currentNestedRouteCount}`
-                    const isCurrentDropdownOpen = this.state[key]
-                    const toggleDropdown = this.toggleDropdown(currentNestedRouteCount)
-                    currentNestedRouteCount += 1
                     return (
-                      <div key={el.name} className={`dropdown  ${isCurrentDropdownOpen ? 'open' : ''} ${this.getBorder(el.name)}`}>
-                        <div onClick={toggleDropdown} className="center-content nav-button-container nav-button hand-cursor"> {displayName} </div>
-                        {
-                          isCurrentDropdownOpen &&
-                            <div className="dropdown-menu">
-                              {
-                                plainRoutes.map(
-                                  (route) => {
-                                    if (route.hideRouteFromNavbar) {
-                                      return false
-                                    }
-                                    const routeDisplayName = route.displayName ? route.displayName : route.name
-                                    return (<Link
-                                      key={route.name}
-                                      onClick={toggleDropdown}
-                                      className="nav-button-item dropdown-item SRC-primary-background-color-hover SRC-nested-color center-content"
-                                      to={route.to!}
-                                    >
-                                      {routeDisplayName}
-                                    </Link>)
-                                  }
-                                )
+                      <Dropdown className={this.getBorder(el.name)}>
+                        <Dropdown.Toggle variant="light" id={displayName} className="center-content nav-button-container nav-button"> 
+                          {displayName} 
+                        </Dropdown.Toggle>
+                        <Dropdown.Menu className="portal-nav-menu">
+                          {
+                            plainRoutes.map(
+                              (route) => {
+                                if (route.hideRouteFromNavbar) {
+                                  return false
+                                }
+                                const routeDisplayName = route.displayName ? route.displayName : route.name
```

Comment:
could this be simplified to   `const routeDisplayName = route.displayName ||  route.name`

Reply:
done